pkg/apis/kops/validation/legacy.go

```diff
@@ -248,11 +248,13 @@ func ValidateCluster(c *kops.Cluster, strict bool) error {
 
 	// AdminAccess
 	if strict && len(c.Spec.SSHAccess) == 0 {
+		// TODO: We may want to allow this
```

Comment:
Because it seems likely to be a mistake.  Really because it started that way, and nobody has ever asked for it to be changed (yet).